tabbycat/draw/generator/bpelimination.py

```diff
@@ -20,7 +20,7 @@ def _four_way_fold(self, teams, start_rank=0):
         pools = (teams[0:n], teams[n:2*n][::-1], teams[2*n:3*n], teams[3*n:4*n][::-1])  # Reverse 1 & 3
         pairings = list()
         for i, ts in enumerate(zip(*pools), start=start_rank+1):
-            pairing = BPPairing(ts, bracket=0, room_rank=i)
+            pairing = PolyPairing(ts, bracket=0, room_rank=i, num_sides=self.teams_in_debate)
```

Comment:
Interesting—in this context wouldn't `num_sides` always be 4? (Since you still have the `len(teams) % 4 == 0` check in line 16?) Why not just hard-code in this case?